File: src/kyu_7/switcheroo/js/switcheroo.js
function switcheroo(x){
    let result = '';
    for (let i = 0; i < x.length; i++) {
        result += x[i] === 'a' ? 'b'
            : x[i] === 'b' ? 'a'
            : x[i];
    }
    return result;
}
module.exports = switcheroo;

File: src/kyu_7/switcheroo/cpp/switcheroo.cpp
#include <string>

#include <string>

std::string switcheroo(const std::string &s)
{
    std::string result;
    for (int i = 0; i < s.length(); i++)
    {
        if(s[i] == 'a')
        {
          result += "b";
        } else if (s[i] == 'b')
        {
          result += "a";
        } else
        {
          result += s[i];
        }
    }
    return result;
}